covid_graphs/setup.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+from setuptools import find_packages, setup
+
+setup(
+    name="covid_graphs",
+    version="1.0",
+    description="COVID-19 graph plotting",
+    install_requires=[
+        "dash",
+        "Flask",
+        "numpy",
+        "pandas",
+        "protobuf",
+        "plotly",
+        "pytest",
+        "scipy",
+
+    ],
+    dependency_links=[],
+    include_package_data=True,
+    python_requires=">=3.7",
```

Comment:
Why do we need 3.7?

Reply:
Using Python 3.7 will be problematic. Travis runs Ubuntu 18.04 and it only has Python 3.6.

Reply:
I thought it would be a problem with Travis, but they support it: https://docs.travis-ci.com/user/reference/bionic/#python-support

Reply:
We definitely want dataclasses and typing for variables. I think 3.7 is the earliest that supports that.